Makefile

```diff
@@ -1,6 +1,7 @@
 BUILD_DIR=$(shell pwd)/build
 CODE_DIR=$(shell pwd)/iso
-VERSION=1.0.0
+VERSIONFILE=VERSION
```

Comment:
-1, I think it just makes things harder. In fact on the minishift side I also want to merge the versions back to the Makfile. This gives you one single source of truth.